proxy/bridge/common.go

```diff
@@ -36,3 +33,27 @@ func GetTlsConfig(log *logrus.Logger) (*tls.Config, error) {
 	tlsConfig.RootCAs = caCertPool
 	return tlsConfig, nil
 }
+
+func GetOIDCTlsConfig() (*tls.Config, error) {
+	tlsConfig := &tls.Config{}
+
+	if config.OIDCInsecure == "true" {
+		log.Warn("Using InsecureSkipVerify for OIDC communication")
+		tlsConfig.InsecureSkipVerify = true
+	}
+
+	_, err := os.Stat("../certs/ca_oidc.crt")
```

Comment:
This cert is for communication with the OIDC (if it uses self-signed cert). It may be different than the Backend one.
I've updated the README.

Note that this change is not reflected in the helm chart yet. I will be creating a separate PR for the deployment updates.